StellarDotnetSdk.Tests/ScValTest.cs

```diff
@@ -341,9 +341,9 @@ public void TestScUint128()
     }
 
     [TestMethod]
-    public void TestScInt128()
+    public void TestScInt128FromParts()
     {
-        var scInt128 = new SCInt128(18446744073709551615, -9223372036854775807);
+        var scInt128 = new SCInt128(-9223372036854775807, 18446744073709551615);
```

Comment:
This is a breaking change I assume?

Reply:
Yes, I've added a note in the PR about that.